src/tests/Microsoft.PowerFx.Interpreter.Tests.Shared/MutationScripts/Coalesce_CoalesceShortCircuit.txt

```diff
@@ -1,3 +1,4 @@
+#SETUP: PowerFxV1CompatibilityRules
```

Comment:
>PowerFxV1CompatibilityRules [](http://example.com/codeflow?start=8&length=27)

Curious: why do we need this? Same question for other test files with this single change

Reply:
So the test cases in these files will return the expected type (table or record).